node-perf-dash/Makefile

```diff
@@ -0,0 +1,18 @@
+all: push
+
+# See pod.yaml for the version currently running-- bump this ahead before rebuilding!
+TAG = v0.1
+
+REPO = choufong
```

Comment:
Instead of your own repo, shouldn't we decide to use gcr.io/google_containers instead?